### PR TITLE
improving doc

### PR DESCRIPTION
**Pull request by CharlyF**
### What does this PR do?

Moving kubernetes section to gain visibility. Adding symlink to increase readability.

### Motivation

Love for Kubernetes.

### CODE REVIEW

**Review by xvello — DISMISSED** [review]
👍

File: Dockerfiles/agent/README.md [hunk 1]
```diff
@@ -77,6 +79,61 @@ To run custom checks and configurations without buidling your own image, you can
...
+In the context of using the Kubernetes integration, and when deploying agents in a Kubernetes cluster, a set of rights are required for the agent to integrate seamlessly.
+
+You will need to allow the agent to be allowed to perform a few actions:
+
+- `get` and `update` of the `Configmaps` named `datadogtoken` to update and query the most up to date version token corresponding to the latest event stored in ETCD.
+- `list` and `watch` of the `Events` to pull the events from the API Server, format and submit them.
+- `get`, `update` and `create` for the `Endpoint`. The Endpoint used by the agent for the [Leader election](#leader-election) feature is named `datadog-leader-election`.
+- `list` the `componentstatuses` resource, in order to submit service checks for the Controle Plane's components status.
+
+You can find the templates in manifests/rbac.
```
Comment (mfpierre) [inline Dockerfiles/agent/README.md]: Could you add a link?

Comment (CharlyF) [reply Dockerfiles/agent/README.md]: yes I can

**Review by mfpierre — APPROVED** [review]
📝

**Pull request merged**